src/libs/OptionsListUtils.js

```diff
@@ -76,6 +77,18 @@ Onyx.connect({
     },
 });
 
+const reportActions = {};
+Onyx.connect({
+    key: ONYXKEYS.COLLECTION.REPORT_ACTIONS,
+    callback: (actions, key) => {
+        if (!key || !actions) {
+            return;
+        }
+        const reportID = CollectionUtils.extractCollectionItemID(key);
+        reportActions[reportID] = _.toArray(actions);
```

Comment:
Since this only needs to keep track of the last report action, how about this? It'll require just a few minor changes below.

```suggestion
        lastReportAction[reportID] = _.last(_.toArray(actions));
```

Reply:
Sounds good! The code looks a bit cleaner below and the `_.last` calculation is made just once

Reply:
One minor thing though, I changed it to `lastReportActions` (plural) because it's a collection, is that ok? Or should it be singular?

Reply:
Yeah, plural is fine.